doc/install/install-ceph-gateway.rst

```diff
@@ -510,24 +510,24 @@ Execute the following steps:
 
 #. Add the following contents to the file::
 
-    import boto
```

Comment:
I think it's superfluous. I checked it btw, and it works. I used pycharm which says it's an unused import.